spiped/main.c

```diff
@@ -198,6 +205,19 @@ main(int argc, char * argv[])
 		}
 	}
 
+	/* Check whether we are init (i.e. running inside a container). */
+	if (getpid() == 1) {
+		/* https://github.com/docker/docker/issues/7086 */
+		warn0("WARNING: Applying workaround for Docker signal-handling bug");
+		/* Bind explicit signal handler for SIGTERM and SIGINT. */
```

Comment:
I'm not a fan of the grammar here either.  s/Bind explicit/Bind an explicit/ perhaps?